data.json

```diff
@@ -6,9 +6,9 @@
             "supportLength": "Extended",
             "releaseDate": "2023-04-26",
             "releaseDateExtra": null,
-            "partialSupport": "2024-04-17",
+            "partialSupport": "2024-04-27",
```

Comment:
The new validation in this PR uncovered a few things that were wrong with data.json